src/main/ngapp/accio/src/app/img-gallery/img-gallery.component.html

```diff
@@ -23,10 +23,74 @@
   <button type="submit" mat-button  id="upload-button">Upload</button>
 </form>
 
-<div id="gallery" *ngIf="displayImages">
+<br>
```

Comment:
NOTE: The forms was being difficult with overlapping, I suspect "display: relative". Didn't want to change css scheme that the UI team had set with consistency/uniformity of the display attribute, so <br> can just be a placeholder if UI team wants to change this overlap correction with css later on when revisiting this directory to implement img download.

Reply:
Instead of these `<br>` s, maybe set the bottom margin of the element above?

Reply:
Noted. Messing with properties of element above gave undesirable css outcomes to the background color. Instead, I will remove the `<br>` and alter the css for the elements below, which have no prior css formatting as they are code introduced in my commit.

Reply:
I will take a look at changing relative. I think it just needs to be consistent over the application. I can also make a break div which spaces the two apart.